addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1945,7 +1945,7 @@ options.registry.WebsiteLevelColor = options.Class.extend({
             color = '';
             gradient = widgetValue;
         } else {
-            color = widgetValue;
+            color = widgetValue === 'false' ? '' : widgetValue;
```

Comment:
I think the real bug here is that you receive `'false'` as `widgetValue` instead of `'', no?` :grimacing: I tested a little and I sometimes receive `''` and sometimes `'false'`. For the trash icon button, I receive `''` on hover and `'false'` when clicked :shrug: Could you check? I think that since it is inconsistent and sometimes send `'false'` and sometimes send `''` as the selected color when none is selected / the trash icon is chosen, we can change that in stable as potential custo should already have relied on it.

To check in 14.0 as well?

Reply:
Root cause identified in 15.0: the inventory of colors also looks at the "Custom" gradient button - which has no color defined and has its `data-color` set to an empty string. This adds a mapping from `""` to `"false"`. Because of this the reset operation's `""` gets turned into `"false"`.
I have removed the "Custom" button from the color inventory.